addons/nuxeo-platform-spreadsheet/nuxeo-platform-spreadsheet-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-spreadsheet/nuxeo-spreadsheet-button.html

```diff
@@ -135,7 +135,7 @@
 
       _close: function() {
         this.$.dialog.toggle();
-        this.nxProvider.fetch();
+        this.fire('document-updated');
```

Comment:
Shouldn't we only do this when the `#save` button on spreadsheet is pressed?

Reply:
The save button does not close the dialog and I don't have any way to tell whether or not things changed so it's safer to just refresh content on close 🤷‍♂

Reply:
😱 👍